defaults.js

```diff
@@ -46,7 +47,20 @@ module.exports = function setDefaults (name, config) {
   if (!config.connections.incoming) {
     // if no incoming connections have been set,
     // populate this with some rad-comprehensive defaults!
-    config.connections.incoming = incomingConnections(config)
+    config.connections.incoming = {
+      net: [{
+	host: config.host || '::',
```

Comment:
this should have one more indent inside the `net: [{...}]` object

Reply:
Thanks. This was resolved in the open PR, #67.